### PR TITLE
Issue #3 and Issue #8

### PR DESCRIPTION
**Pull request by PriyaBihani**
## Please, go through these steps before you submit a PR.

- [x] My Pod Leader knows I'm working on this Pull Request
- [x] I've explained what the Pull Request is adding.
- [x] I've explained why this is important.

### CODE REVIEW

**Comment by PriyaBihani** [comment]
In this PR we are trying to solve #3 and #8 issue.

File: Gemfile.lock [hunk 1]
```diff
@@ -53,13 +55,22 @@ GEM
     safe_yaml (1.0.5)
     sassc (2.4.0)
       ffi (~> 1.9)
```
Comment (grace-omotoso) [inline Gemfile.lock]: Please, remove gemlock file changes

**Review by grace-omotoso — CHANGES_REQUESTED** [review]
There is a bit of regression here. The project listings have been distorted. Also, device a way to include all the fellows. You don't have to show all of them at once but we can see 3 fellows at a time.

**Comment by grace-omotoso** [comment]
@PriyaBihani and co. I am still seeing the regression with the project card. You can compare the project repository on the main branch with what you sent out in the PR. You will see that the project cards do not appear the same way. If you still cannot see the difference, we can have a call together.

**Comment by PriyaBihani** [comment]
> @PriyaBihani and co. I am still seeing the regression with the project card. You can compare the project repository on the main branch with what you sent out in the PR. You will see that the project cards do not appear the same way. If you still cannot see the difference, we can have a call together.

@grace-omotoso We are working on it. I will contact you, if I don't understand anything.

**Comment by AdarshNaidu** [comment]
@grace-omotoso please review!! We will ask pod mates to update their respective details once this is merged!

**Comment by grace-omotoso** [comment]
Great, project cards are no longer distorted. However, I have a concern with displaying all the pod members at once. Would we be doing that if we had 100 pod members? Please, use a carousel instead. Thanks

**Comment by PriyaBihani** [comment]
@grace-omotoso Added the carousel. Can you check it out?

**Comment by grace-omotoso** [comment]
@PriyaBihani, it is a lot better, please, fix merge conflicts.

**Comment by grace-omotoso** [comment]
@PriyaBihani Can we have the location also displayed on the card e.g Murfreesboro, TN, USA. No need to add long and lat

**Comment by PriyaBihani** [comment]
@grace-omotoso Fixed the merge conflicts and added location, please review.

File: _sass/description.scss [hunk 2]
```diff
@@ -0,0 +1,132 @@
+.fellow-container-heading {
+	font-size: 2.4rem;
+}
+
+.fellow-cards-container {
+	// display: flex;
+	// justify-content: center;
+	// align-items: center;
+	// min-height: 70vh;
+
```
Comment (grace-omotoso) [inline _sass/description.scss]: remove commented code

File: _sass/description.scss [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+	.fellow-container {
+		// position: relative;
+		// width: 1280px;
+		// display: flex;
+		// justify-content: center;
+		// align-items: center;
+		// flex-wrap: wrap;
```
Comment (grace-omotoso) [inline _sass/description.scss]: remove commented code

**Review by grace-omotoso — APPROVED** [review]

**Pull request merged**